package.json

```diff
@@ -119,7 +119,7 @@
     "react-native-key-command": "^1.0.1",
     "react-native-localize": "^2.2.6",
     "react-native-modal": "^13.0.0",
-    "react-native-onyx": "1.0.52",
+    "react-native-onyx": "git+https://github.com/fabioh8010/react-native-onyx#86aa5492d7013b59bcb34d015963cc4808651e26",
```

Comment:
we actually have to change this to the actual Onyx version after [this PR](https://github.com/Expensify/react-native-onyx/pull/267) is merged and the version is bumped and the new version is published to npm.

but yes, we have to wait for [this PR](https://github.com/Expensify/App/pull/24041) before doing that